### PR TITLE
Resolution for Issue #10

### PR DESCRIPTION
**Pull request by simonwgill**
Changed the download to check currently configured sources for nuget.commandline before falling back to the default Microsoft link.

### CODE REVIEW

File: NuGetMSBuild.psm1 [hunk 1]
```diff
@@ -32,7 +32,15 @@ function Ensure-NuGetBuild {
     $nugetToolsPath = (Join-Path $solutionDir .nuget)
     
     if(!(Test-Path $nugetToolsPath) -or !(Get-ChildItem $nugetToolsPath)) {
-        Install-Package NuGet.Build -Source 'https://go.microsoft.com/fwlink/?LinkID=206669'
+        Write-Host "Tool path does not exist or could not find nuget.exe and msbuild scripts in the tool path."
+
+	try {
+	        Install-Package NuGet.Build
```
Comment (davidfowl) [inline NuGetMSBuild.psm1]: Instead of Try Catch, do, Get-Package -List NuGet.Build to see if it exists.

**Comment by davidfowl** [comment]
Any updates?

**Comment by simonwgill** [comment]
Yes, there's a few changes from the 17th. Github is still new to me, so I guessed that you would see them when you looked again. Looks like that wasn't right ;)

**Comment by Mystere** [comment]
Could we get this merged into the main branch and a new package deployed?

**Comment by davidfowl** [comment]
Don't think we need to. This feature is baked into nuget itself.

**Comment by Mystere** [comment]
That doesn't change the fact that current version of this package is broken.  Maybe this feature is built in, but it would solve the problem.

**Comment by davidfowl** [comment]
I can live with that

Sent from my Windows Phone

**Pull request closed without merging**